### PR TITLE
Pairprog

### PR DESCRIPTION
**Pull request by Tofcookies**
## Portfolio: contact-form;
**In this project, I;**

Added a form tag, input field, textarea and button  to my index.html file
Styled the  input field, textarea and button in the style.css
Ensured there were no linter errors

This is my contact form in the portfolio mobile version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,17 +67,174 @@ <h2>I'm a software developer</h2>
...
     </section>
+    <form class="form" action="https://formspree.io/f/mzbyokvj" method="post">
+      <h1 class="formHeading">I'm always interested in hearing about new projects, so if you'd like to chat please get
+        in touch.</h1>
+      <input type="text" id="name" name="name" placeholder="Full name" class="nameText" minlength="1" maxlength="30"
+        required />
+      <input type="email" id="email" name="email" placeholder="Email address" class="emailText" minlength="1"
+        required />
+      <textarea name="text" minlength="1" maxlength="500">Enter Your text here</textarea>
+      <button class="click" type="submit">Get in touch</button>
```
Comment (SajjadAhmad14) [inline index.html:236]: Please implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state).

File: index.html [hunk 2]
```diff
@@ -67,17 +67,174 @@ <h2>I'm a software developer</h2>
...
+      </div>
     </section>
+    <form class="form" action="https://formspree.io/f/mzbyokvj" method="post">
+      <h1 class="formHeading">I'm always interested in hearing about new projects, so if you'd like to chat please get
+        in touch.</h1>
+      <input type="text" id="name" name="name" placeholder="Full name" class="nameText" minlength="1" maxlength="30"
+        required />
+      <input type="email" id="email" name="email" placeholder="Email address" class="emailText" minlength="1"
+        required />
+      <textarea name="text" minlength="1" maxlength="500">Enter Your text here</textarea>
```
Comment (SajjadAhmad14) [inline index.html]: Please add the `required` attribute to activate validations for textarea.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tof-tof Alugbin, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Tofcookies , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing the requested changes, but your linters are failing, kindly fix that and resubmit for review. 
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

- [ ] Please you linter (Stylelint) checks are not passing, kindly fix that an resubmit for review, see image.
stylelint

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @Tofcookies , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**